ara/filters.py

```diff
@@ -26,9 +26,14 @@ def jinja_time_formatter(timestamp):
 
     @app.template_filter('to_nice_json')
     def jinja_to_nice_json(result):
-        """ Formats a result """
-        return json.dumps(result, indent=4, sort_keys=True,
-                          default=str)
+        """ Tries to formats a result as a pretty printed JSON. """
+        try:
+            return json.dumps(json.loads(result), indent=4, sort_keys=True)
+        except:
```

Comment:
I would rather catch explicit exceptions here.  I think that if we catch `ValueError` and `TypeError` we'll be okay.  Otherwise this could break in mysterious ways (for example, an unadorned `except` will trap syntax errors, which is a pain).